README.md

```diff
@@ -220,7 +220,7 @@ When using React Test Utils with this library, here are some things to keep in m
 - You need to use the `.portal` property, as in `ReactDOM.findDOMNode(renderedModal.portal)` or `TestUtils.scryRenderedDOMComponentsWithClass(Modal.portal, 'my-modal-class')` to acquire a handle to the inner contents of your modal.
 
 By default the modal is closed when clicking outside of it (the overlay area). If you want to prevent this behavior you can
```

Comment:
Hmm... maybe this whole section should read more like:

```
By default the modal will attempt to close when clicking outside of it (on the overlay area).  Because the open and closed state of the modal should be controlled outside of the modal, you need to provide the `onRequestClose` function prop to actually set the state.  If you want to prevent the overlay close behavior, you should pass the `shouldCloseOnOverlayClick` props as `false`.